plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -224,6 +228,7 @@ func (p *pusher) send() {
 	for {
 		input.SequenceToken = p.sequenceToken
 		output, err := p.Service.PutLogEvents(input)
+		p.Log.Infof("calling put log events %v", p.Group)
```

Comment:
Considering the volume of this call, maybe not a good idea to log every put log events? I'm afraid this could flood the agent log, or maybe make it debug level?

Reply:
thanks, I put these as i was debugging, deleting them now